examples/example_composed_target_function.py

```diff
@@ -0,0 +1,160 @@
+"""
+Example for evolutionary regression on a composed target function
```

Comment:
Instead of "composed target function", I would say "piecewise target function". Also in other comments below.